src/notation/internal/notationinteraction.cpp

```diff
@@ -556,6 +556,8 @@ void NotationInteraction::clearSelection()
     score()->deselectAll();
 
     notifyAboutSelectionChanged();
+
+    selection()->setHitElementContext(INotationSelection::ElementContext());
```

Comment:
Maybe better add method like `resetElementContext`

Reply:
No more need, because setHitElementContext is a part of NotationInteraction now